tests_tf/test_attacks.py

```diff
@@ -1018,6 +970,26 @@ def test_multiple_initial_random_step(self):
     pass
 
 
+class TestMomentumIterativeMethod(TestBasicIterativeMethod):
```

Comment:
It would be better if this class didn't move, so that the git history is less confusing

Reply:
Done, but git diff still gets confused.